src/scss/02-layout/_layout.scss

```diff
@@ -173,14 +180,26 @@ body,
 // IsRTL -------------------------------------------------------------------------
 ///
 .is-rtl {
+	.app-menu-content {
+		left: auto;
+		right: calc(-1 * var(--side-menu-size));
+	}
+
+	.menu-visible {
+		.app-menu-content {
+			right: 0;
+			-webkit-transform: translateX(0) translateZ(0);
+			transform: translateX(0) translateZ(0);
+		}
+	}
+
```

Comment:
These selectors should be placed inside **_menu.scss** partial not inside **_layout.scsc** one.

Reply:
Done